### PR TITLE
Make singular strings singular

### PR DESCRIPTION
**Pull request by eloquence**
(Thanks to AO for pointing this out via Weblate comments)

## Status

Ready for review

### CODE REVIEW

File: securedrop/journalist_app/forms.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def minimum_length_validation(form: FlaskForm, field: Field) -> None:
     if len(field.data) < Journalist.MIN_USERNAME_LEN:
         raise ValidationError(
             ngettext(
-                'Must be at least {num} characters long.',
+                'Must be at least {num} character long.',
```
Comment (eloquence) [inline securedrop/journalist_app/forms.py:31]: Could also get rid of `{num}` here and say `1` or `one`, not sure if any option is better from the standpoint of translators

Comment (rmol) [reply securedrop/journalist_app/forms.py:31]: I don't know. The only benefit I saw of replacing the placeholder was that `one` meets AP style rules, but we still fall afoul of them at `2`. :wink: 

Either option right now can cause a Weblate check complaint, so no benefit there. I would just go with what you have.

**Comment by conorsch** [comment]
For context, see https://weblate.securedrop.org/translate/securedrop/securedrop/en/?checksum=18c0a71ef2d0d92b#comments

**Review by conorsch — APPROVED** [review]
Approving based on visual review. Thanks for prompt turnaround, both!

**Comment by conorsch** [comment]
Fine to merge once tests pass. @creviera I suggest we proactively open a backport PR into release/1.7 so we can run CI concurrently. Agree?

**Pull request merged**